packages/core/src/node/messaging/ipc-connection-provider.ts

```diff
@@ -92,8 +92,8 @@ export class IPCConnectionProvider {
         }
 
         const childProcess = cp.fork(path.resolve(__dirname, 'ipc-bootstrap.js'), options.args, forkOptions);
-        childProcess.stdout.on('data', data => this.logger.info(`[${options.serverName}: ${childProcess.pid}] ${data.toString()}`));
-        childProcess.stderr.on('data', data => this.logger.error(`[${options.serverName}: ${childProcess.pid}] ${data.toString()}`));
+        childProcess.stdout.on('data', data => this.logger.info(`[${options.serverName}: ${childProcess.pid}] ${data.toString().trim()}`));
```

Comment:
Should not we try to preserve the original output?

Reply:
Did you mean the `trim`? Otherwise, we end up having an additional empty new line after each IPC message.

Reply:
yes, i see, ok